index.html

```diff
@@ -20,21 +20,21 @@
         <!-- navbar-section -->
         <header class="header-section">
           <span class="nav-span"><a href="#">Promise Okechukwu</a></span>
-          <span><a href="#"><img src="./images/Enabled.png" class="#" alt="#"></a></span>
+          <span><a href="#"><img src="./images/Enabled.png" class="menubtn" alt="#"></a></span>
           <ul class="nav-links">
             <li>
               <span><a href="#">Portfolio</a></span>
               <span><a href="#">About</a></span>
-              <span><a href="#">contact</a></span>
-              <span><a href="#"><img src="./images/mail.png" alt="mail logo"></a></span>
+              <span><a href="#">Contact</a></span>
+              <span><a href="#"><img src="images/Union.png" alt="mail logo"></a></span>
             </li>
           </ul>
         </header>
         <!-- header  section -->
         <section class="first-section">
           <div class="header-intro">
             <h1 class="header-text">
-              Hey There.
+              Hey There. <br>
               I’m Promise
             </h1>
             <h2 class="header-smaller-text"> I am a Software Developer </h2>
```

Comment:
- [ ] Great job so far just a small update here please align the header in the center to match the Figma design🙏🏻✅

- Your design 
![image](https://user-images.githubusercontent.com/58553711/177674884-1047d5ae-cf2e-46a5-b9aa-680ad621688f.png)

- Figma design
![image](https://user-images.githubusercontent.com/58553711/177674938-48aa6a55-814b-4474-bd7a-67851374e585.png)